src/angular/projects/spark-core-angular/src/lib/components/sprk-button/sprk-button.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
```

Comment:
I thought we changed button to be directive? That might be in a diff PR tho.

Reply:
we did, i think you had the change and I didnt, when i merged i got both, i'll clean it up